Mathlib/Analysis/CStarAlgebra/Module/Defs.lean

```diff
@@ -27,13 +27,26 @@ Cauchy-Schwarz inequality, and induces a norm that makes `E` a normed vector spa
 
 ## Implementation notes
 
-The class `CStarModule A E` requires `E` to already have a `Norm E` instance on it, but
-no other norm-related instances. We then include the fact that this norm agrees with the norm
-induced by the inner product among the axioms of the class. Furthermore, instead of registering
-`NormedAddCommGroup E` and `NormedSpace ℂ E` instances (which might already be present on the type,
-and which would send the type class search algorithm on a chase for `A`), we provide a
-`NormedSpace.Core` structure which enables downstream users of the class to easily register
-these instances themselves on a particular type.
+Unlike the vast majority of other classes in Mathlib, notably `InnerProductSpace`, the class
+`CStarModule A E` does *not* take a `Norm E` instance as an argument, nor does it extend `Norm E`.
```

Comment:
I don't understand this -- it *does* take a `[Norm E]` argument...

Reply:
sorry this was written from when I had temporarily changed it, before deciding it was a bad idea, and then I failed to catch it because git successfully auto-merged master.